tests/cold/test_ca_crt_migration.py

```
@@ -0,0 +1,166 @@
+class TestCaCrtColdMigration:
+    """Cold migration using ca.crt secret field instead of cacert (MTV-4561)."""
+
+    storage_map: StorageMap
+    network_map: NetworkMap
+    plan_resource: Plan
+
+    def test_create_storagemap(
+        self,
+        prepared_plan: dict[str, Any],
+        fixture_store: dict[str, Any],
+        ocp_admin_client: "DynamicClient",
```

Ruff (0.15.18)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)